test/fixtures/cli-plugin-local/.gitattributes

```diff
@@ -0,0 +1 @@
+*.md text=auto eol=lf
```

Comment:
Why was this change needed? (Sorry, don't have access to a Windows computer right now that I can test this on.) Were the line endings on the file being output by Metalsmith somehow different from the input file?